packages/grpc-native-core/index.d.ts

```diff
@@ -302,13 +308,13 @@ declare module "grpc" {
   /**
    * User provided method to handle client streaming methods on the server.
    */
-  type handleClientStreamingCall = (call: ServerReadableStream, callback: sendUnaryData) => void;
+  type handleClientStreamingCall<T> = (call: ServerReadableStream<T>, callback: sendUnaryData<T>) => void;
 
   /**
    * A stream that the server can read from. Used for calls that are streaming
    * from the client side.
    */
-  export class ServerReadableStream extends Readable {
+  export class ServerReadableStream<T> extends Readable {
```

Comment:
I assume that `T` is supposed to be the request type here, but because it's not used, it's not entirely clear.

Reply:
The hope was that one could later introduce type specific stream class definitions that refine the existing ones from node, without having another API breakage.

Reply:
For consistency with the other call types, this should probably be parameterized on `RequestType`, even if you don't use the type in the definition.